ndmg/stats/qa_reg.py

```diff
@@ -213,16 +235,35 @@ def plot_overlays(atlas, b0, cmaps=None, minthr=2, maxthr=95, edge=False):
                 image = edge_map(image).data
                 image[image > 0] = max_val
                 image[image == 0] = min_val
-
+            #Set the axis invisible
+            plt.xticks([])
+            plt.yticks([])
+            
+            #Set the frame invisible
+            ax.spines['top'].set_visible(False)
+            ax.spines['right'].set_visible(False)
+            ax.spines['bottom'].set_visible(False)
+            ax.spines['left'].set_visible(False)
+            
             ax.imshow(atl, interpolation="none", cmap=cmaps[0], alpha=0.9)
             ax.imshow(
                 opaque_colorscale(
                     cmaps[1], image, alpha=0.9, vmin=min_val, vmax=max_val
                 )
             )
-
+            
+            #set the legend
+            if idx == 3:
+                plt.plot(0,0,"-",c="pink",label='registered')
+                plt.plot(0,0,"-",c="green",label='reference')
+                plt.legend(loc='best',fontsize=12,bbox_to_anchor=(1.5,1.5))
+            
+    #Set title for the whole picture
+    [a,b,c] = shap
+    title = 'QA For Registration\n\nVolume:'+ str(a) +'*'+ str(b) + '*' + str(c)+'\n'
+    foverlay.suptitle(title,fontsize=24)
```

Comment:
- can just be `a, b, c = shap`
- `shap` is a confusing name since it's not a word. Do you mean `shape`? (even that is a bit confusing, but less so)
-

Reply:
see https://github.com/neurodata/ndmg/pull/372/commits/0d0f1fcea693deac7062a337bbabd64c3262d7a7

I changed the variable "shap" to "brain_volume" to be specific.
This commit contains not just the change of variable name "shap". I'm sorry to trouble you with finding the changes